pkgs/by-name/bl/bluez/package.nix

```
@@ -26,6 +25,16 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-SZ1/o0WplsG7ZQ9cZ0nh2SkRH6bs4L4OmGh/7mEkU24=";
   };
 
+  patches =
+    # Disable one failing test with musl libc, also seen by alpine
+    # https://github.com/bluez/bluez/issues/726
+    lib.optional (stdenv.hostPlatform.isMusl && stdenv.hostPlatform.isx86_64 && finalAttrs.doCheck)
```

```suggestion
    lib.optional (stdenv.hostPlatform.isMusl && stdenv.hostPlatform.isx86_64)
```

Doesn't really make sense to have two different sets of patches for x86_64 musl depending on whether tests are enabled, right?